### DISCUSSION TITLE
Player Cards Still Show On SignOut

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#223] [Help Me!] [answered]
Dani Crosby

Have you searched for other help tickets? Yes

Describe your issue

In the Sports Roster assignment, I am able to sign in and sign out but my player cards are still on the page when I sign out. My links hide, but not the page itself.

What have you tried? 
My group has tried to tweak the routes, nav, uid, Watched videos, Timeboxing 30 mins.

https://www.pluralsight.com/guides/how-to-set-react-router-default-route-redirect-to-home
https://reactnavigation.org/docs/navigating/
https://reactrouter.com/web/api/Switch

Files in question: 
https://github.com/danicrosby/sports-roster/tree/signout-bug-fix

Give me a comment below letting us know that you understand these items or let us know in the comments if you have any questions.

Got it!

**Comment by danicrosby** [comment, marked as answer]
Possible fix: Redirect with Private Route: 

https://reactrouter.com/web/api/Switch

**Reply by drteresavasquez** [reply]
Yes. If it's a private route, it will redirect the user on logout but you have to make sure you're watching the user state.

**Reply by drteresavasquez** [reply]
@danicrosby look at the May 3 info and check out the private routes video. https://github.com/nss-evening-cohort-14/evening-client-side/discussions/218

**Comment by drteresavasquez** [comment]
@danicrosby was this resolved?
